wp-content/themes/workingnyc/views/search/search-result.vue

```diff
@@ -0,0 +1,16 @@
+<template>
```

Comment:
I have added to this file and search-results-list.vue to make search results show up properly in the next couple of PRs.